docs/sources/reference/commandline/cli.md

```diff
@@ -364,6 +364,22 @@ flag to the Docker daemon as described above.
 Local registries, whose IP address falls in the 127.0.0.0/8 range, are automatically marked as insecure
 as of Docker 1.3.2. It is not recommended to rely on this, as it may change in the future.
 
+### Running a Docker daemon behind a HTTPS_PROXY
+
+When running inside a LAN that uses a `HTTPS` proxy, the Docker Hub certificates
+will be replaced by the proxy's certificates. These certificates need to be added
+to your Docker host's configuration:
```

Comment:
Passive voice is a confusing here. Suggest something like "When running inside a LAN that uses a `HTTPS` proxy, you will need to replace the Docker Hub certificates with the proxy's certificates. Follow these steps to configure your Docker host to use the proxy certificates:"